gr-digital/lib/additive_scrambler_bb_impl.h

```diff
@@ -21,29 +21,29 @@ class additive_scrambler_bb_impl : public additive_scrambler_bb
 {
 private:
     digital::lfsr d_lfsr;
-    int d_count; //!< Reset the LFSR after this many bytes (not bits)
-    int d_bytes; //!< Count the processed bytes
-    int d_len;
-    int d_seed;
-    int d_bits_per_byte;
+    int64_t d_count;  //!< Reset the LFSR after this many bytes (not bits)
+    uint64_t d_bytes; //!< Count the processed bytes
+    uint8_t d_len;
+    uint64_t d_seed;
+    uint8_t d_bits_per_byte;
     pmt::pmt_t d_reset_tag_key; //!< Reset the LFSR when this tag is received
 
-    int _get_next_reset_index(int noutput_items, int last_reset_index = -1);
+    int64_t _get_next_reset_index(int64_t noutput_items, int64_t last_reset_index = -1);
 
 public:
-    additive_scrambler_bb_impl(int mask,
-                               int seed,
-                               int len,
-                               int count = 0,
-                               int bits_per_byte = 1,
+    additive_scrambler_bb_impl(uint64_t mask,
+                               uint64_t seed,
+                               uint8_t len,
+                               int64_t count = 0,
+                               uint8_t bits_per_byte = 1,
                                const std::string& reset_tag_key = "");
     ~additive_scrambler_bb_impl() override;
 
-    int mask() const override;
-    int seed() const override;
-    int len() const override;
-    int count() const override;
-    int bits_per_byte() override { return d_bits_per_byte; };
+    uint64_t mask() const;
```

Comment:
is there a reason we're removing the `override` here and elsewhere?

Reply:
No, this is probably a typo, I didn't think of it when writing, but it should be there I think! Will update.